amgut/handlers/human_survey_completed.py

```diff
@@ -19,6 +80,17 @@ def get(self):
 
             self.render('human_survey_completed.html', skid=self.current_user,
                         surveys=surveys)
+            consent_info = ag_data.getConsent(human_survey_id)
+            message = build_consent_form(consent_info)
+
+            try:
+                send_email(message, 'American Gut - Signed Consent Form(s)',
+                           recipient=consent_info['participant_email'])
+            except:
+                Logger.exception('Error sending signed consent form for '
+                                 'survey ID: %s to email: %s' %
+                                 (human_survey_id,
+                                  consent_info['participant_email']))
```

Comment:
Testing on my system, I got an exception when it tried to send an email. However, this call to `Logger.exception` failed with this error message:
```
  File "/Users/adam/.virtualenvs/python276/lib/python2.7/site-packages/tornado/web.py", line 1346, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/Users/adam/.virtualenvs/python276/lib/python2.7/site-packages/tornado/web.py", line 2626, in wrapper
    return method(self, *args, **kwargs)
  File "/Users/adam/git/american-gut-web/amgut/handlers/human_survey_completed.py", line 94, in get
    consent_info['participant_email']))
TypeError: unbound method exception() must be called with Logger instance as first argument (got str instance instead)
```